thirdeye/thirdeye-frontend/app/pods/custom/composite-animalies-table/resolution/component.js

```diff
@@ -0,0 +1,18 @@
+import Component from "@ember/component";
+import { set } from '@ember/object';
+
+export default Component.extend({
+    didReceiveAttrs() {
+        this._super(...arguments);
+    },
+    
+    actions: {
+        onChangeAnomalyResponse: (anomalyObject, selection, options ) => {
```

Comment:
I am not sure I fully understand the function. This function when being referenced from the template is passing in just 1 parameter - `record` which I assume is the first argument in this function. Where do `selection` and `options` come from? Is it automatically added by `power-select`? Also, can you please add comments for this action in the format mentioned in one of my comments above?

Reply:
this function is currently just a placeholder for now.
  `selection` is what the user selected onChange
  `options` is the available in the dropdown.
Yes, they are provided by `power-select`